deps/pelib/CMakeLists.txt

```diff
@@ -8,8 +8,7 @@ if(CMAKE_CXX_COMPILER)
 endif()
 
 ExternalProject_Add(pelib-project
-	URL https://github.com/avast-tl/pelib/archive/e93eaa7c150f4608a5d02a67f5edc9e54456fe24.zip
-	URL_HASH SHA256=2ffd7e89451c980a1af6d24d4f6dfbb69a660b06ad5de44c481f6431e21de394
+	URL https://github.com/avast-tl/pelib/archive/master.zip
```

Comment:
Please do not use branches as reference points. Branches can change over time. Use only commit hashes or tags. Also please add missing `URL_HASH`.

Reply:
Fixed in commit cab945158749486bd6a92ced84237678963a1291